### ISSUE TITLE
Installing (actually compiling) it

### ISSUE THREAD

**Issue by Juanjdurillo** [KhaosResearch/MORPHY#1]
Hi, 

I tried to install the software as indicated in a machine running Ubuntu 16.04 following the indicated steps (sh install.sh) but I run into several troubles. 

First one, which is not commented, the install script should be run with root permissions (e.g., with sudo), since it tries to modify /usr/local/lib

Second, the script fails in line 20 (fi comparison): replace the '[[' by '[' and ']]' by ']'

Once solved these two issues, I run into a third one: pll/pll.h library is not found, indicating that there are some problems which the header locations. I only found a pll.h header file under lib/Pll/src/ but it is not contained within a pll folder. Therefore, in order to avoid the problem I modified the makefile for indicating the aforementioned folder as a searchable path for the include and I also modified src/problems/./Phylogeny.h line 84, #include <pll.h> instead of #include <pll/pll.h>

After all these modifications, the software is still not installed on my machine:

Compiling  src/main/MORPHY.cpp
g++ -O3  src/main/mainMORPHY.cpp lib/libjmetal.a -o bin/MORPHY -I src/core/. -I src/encodings/. -I src/encodings/variable/. -I src/encodings/solutionType/. -I src/main/. -I src/metaheuristics/. -I src/operators/. -I src/operators/mutation/. -I src/operators/selection/. -I src/problems/. -I src/util/. -I src/util/archive/. -I src/util/comparators/. -I /home/juan/Documents/antonio/prueba-khaos/MORPHY/lib/Pll/src/ -lbpp-core -lbpp-seq -lbpp-phyl -lpll-sse3 -lm  -lm 
/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libpll-sse3.so: undefined reference to `isGap'
collect2: error: ld returned 1 exit status
Makefile:118: recipe for target 'MainMORPHY_main' failed
make: **\* [MainMORPHY_main] Error 1

**Comment by cristianzambrano** [comment]
Hi Juanjo

A new Makefile.in file of PLL package has been added. The default instalation paths are /usr/local/lib and /usr/local/include, so I think is not necessary specify the -I parameter.
So, the changes on _src_ folder also are not necessary, the path of the Pll Include files is /usr/local/include/pll/

**Comment by Juanjdurillo** [comment]
Hi Christian, 
I pulled the new makefile and gave it a try. It didn't compile (a broken installation from the previous try, I though). 

Then I removed everything (a clean option that cleans more than simply the jmetal.a library would be great) and clone the repository again. These are the two errors I got: 
- When installing PLL

configure.ac:25: error: required file './ltmain.sh' not found
src/Makefile.am:8: warning: variable 'libpll_generic_a_SOURCES' is defined but no program or
src/Makefile.am:8: library has 'libpll_generic_a' as canonical name (possible typo)
src/Makefile.am:7: warning: variable 'libpll_generic_la_LDFLAGS' is defined but no program or
src/Makefile.am:7: library has 'libpll_generic_la' as canonical name (possible typo)
Makefile:244: recipe for target 'Makefile.in' failed
make: **\* [Makefile.in] Error 1
-e Compilation error Pll.
- When installing MORPHY

/usr/lib/gcc/x86_64-linux-gnu/5/../../../../lib/libpll-sse3.so: undefined reference to `isGap'
collect2: error: ld returned 1 exit status
Makefile:118: recipe for target 'MainMORPHY_main' failed
make: **\* [MainMORPHY_main] Error
